sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2022-2023.bpmn

```diff
@@ -596,17 +596,17 @@
     <bpmn:sequenceFlow id="Flow_15xhu93" sourceRef="Event_0rpww4j" targetRef="Event_1mkiqso" />
     <bpmn:sequenceFlow id="Flow_1idr6ms" sourceRef="Event_1mkiqso" targetRef="Event_0hxxmu9" />
     <bpmn:textAnnotation id="TextAnnotation_1myp6tg">
-      <bpmn:text>Use CRA Income if provided
-partner1CalculatedTotalIncome
+      <bpmn:text>Use CRA Income if provided
+partner1CalculatedTotalIncome
 studentCalculatedTotalIncome</bpmn:text>
     </bpmn:textAnnotation>
     <bpmn:textAnnotation id="TextAnnotation_0o432k6">
-      <bpmn:text>Naming Conventions
-offeringX (comes directly from offering)
-studentDataX (comes from student application)
-parent1X (comes from parent 1 form)
-parent2X (comes from parent 2 form)
-partner1X (comes from partner form)
+      <bpmn:text>Naming Conventions

```

Comment:
The `parttime-assessment-*` files are a close copy and paste from each other for now, please ensure that the previous year also has these changes.

Reply:
This happened when I switched from CRLF to LF and saved the file. It came from the vs setting and extension. It only affects those three files, and others are unchanged.